crates/fluvio-version-manager/src/common/fluvio.rs

```diff
@@ -0,0 +1,57 @@
+use std::fs::create_dir;
+use std::path::PathBuf;
+
+use anyhow::Result;
+
+use super::home_dir;
+
+/// Home Directory for the Fluvio installation
+pub const FLUVIO_HOME_DIR: &str = ".fluvio";
+
+/// Retrieves the path to the `~/.fluvio` directory in the host system
+pub fn fluvio_path() -> Result<PathBuf> {
+    Ok(home_dir()?.join(FLUVIO_HOME_DIR))
```

Comment:
The Fluvio path can be overridden by env var in the current installer.  We should preserve that behavior